RxSwift/Schedulers/Internal/ScheduledItem.swift

```diff
@@ -6,21 +6,21 @@
 //  Copyright © 2015 Krunoslav Zaher. All rights reserved.
 //
 
-struct ScheduledItem<T>
+struct ScheduledItem<Element>
```

Comment:
```suggestion
struct ScheduledItem<State>
```

Reply:
Changed to `<State>` instead of `<Element>` as suggested!